src/components/treebeard/ScriptsTree.jsx

```diff
@@ -97,7 +97,7 @@ class ScriptsTree extends React.Component {
   }
 
   render() {
-    if (this.state.data === []) {
+    if (this.state.data.length === 0) {
```

Comment:
why is this better?

Reply:
Since this.state.data === [] compares the references of two objects, it will always evaluate to false

Reply:
Comparing arrays using === checks if the two references point to the same object, not if their contents are the same. So it always evaluates to false.

Reply:
ok, i wrote it because I saw that u started to assume that this.state
data is always an array and cannot be null, which is most likely fine i just wanted to check